third-party/realdds/include/realdds/dds-stream-server.h

```diff
@@ -49,10 +52,9 @@ class dds_stream_server : public dds_stream_base
     void close();
 
     bool is_streaming() const override { return _streaming; }
-    void start_streaming( const image_header & header );
+    void start_streaming( const image_header & );
```

Comment:
Why did you remove the variable name?

Reply:
Because saying `image_header header` is redundant and unhelpful. If the type conveys the meaning, then everything else is extra. But that's in hindsight -- maybe I changed something and then undid it...?